istioctl/cmd/istioctl/authn.go

```diff
@@ -46,11 +48,20 @@ istioctl authn tls-check 656bd7df7c-5zp4s.default bar
 				return err
 			}
 			podName, ns := inferPodInfo(args[0], handleNamespace())
-			debug, err := kubeClient.PilotDiscoveryDo(istioNamespace, "GET",
+			results, err := kubeClient.AllPilotsDiscoveryDo(istioNamespace, "GET",
 				fmt.Sprintf("/debug/authenticationz?proxyID=%s.%s", podName, ns), nil)
 			if err != nil {
 				return err
 			}
+			var debug []v2.AuthenticationDebug
+			for i := range results {
```

Comment:
Just want to be sure, when there are multiple pilot instances, what does the `results` response look like? Only one of it include the `v2.AuthenticationDebug`?

Reply:
I checked the raw output and one has the data and the rest have an empty json array.